test/api/workflows_format_2/converter.py

```diff
@@ -1,14 +1,13 @@
-import os
-import sys
+"""Functionality for converting a Format 2 workflow into a standard Galaxy workflow."""
+from __future__ import print_function
 
-import yaml
+from collections import OrderedDict
```

Comment:
Now that I look at the 3 side by side, I think I like smarkets best.  Second choice would be cryptography.